pkg/controller/awsprivatelink/awsprivatelink_controller.go

```diff
@@ -301,7 +299,13 @@ func (r *ReconcileAWSPrivateLink) Reconcile(ctx context.Context, request reconci
 		return reconcile.Result{}, nil
 	}
 
-	return r.reconcilePrivateLink(cd, &hivev1.ClusterMetadata{InfraID: *cp.Spec.InfraID, AdminKubeconfigSecretRef: *cp.Spec.AdminKubeconfigSecretRef}, logger)
+	return r.reconcilePrivateLink(
+		cd,
+		&hivev1.ClusterMetadata{
+			InfraID:                  *cp.Spec.InfraID,
+			AdminKubeconfigSecretRef: *cp.Spec.AdminKubeconfigSecretRef, // HIVE-2585: via ClusterMetadata
```

Comment:
I don't have perms to view this issue but just wanted to ensure this comment is intentional

Reply:
> I don't have perms to view this issue

You do now. But that does put rather an ick on the posterity argument. Then again, hopefully someone whacking moles in this area later would be given access :shrug:

Reply:
D'oh, the comment has a typo here.
```suggestion
			AdminKubeconfigSecretRef: *cp.Spec.AdminKubeconfigSecretRef, // HIVE-2485: via ClusterMetadata
```

Reply:
I didn't catch it earlier but yeah, this is a typo - 2485 not 2585